### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [ ] I added unit tests for new functionality or bug fixes.
* [ ] I added integration tests for new functionality or bug fixes.
* [ ] I added compatibility tests for new functionality or bug fixes.
* [x] I made spot refactorings.
* [x] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [x] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by Fashander — APPROVED** [review]

**Review by chilagrow — DISMISSED** [review]

**Review by chilagrow — DISMISSED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM overall, added a minor question about docs.

File: website/docs/flags.md [hunk 1]
```diff
@@ -7,34 +7,34 @@ FerretDB provides numerous configuration flags that you can customize to suit yo
 You can always access them by using `--help` flag.
```
Comment (b1ron) [inline website/docs/flags.md:7]:
```suggestion
You can always access them by using the `--help` flag.
```

**Review by b1ron — CHANGES_REQUESTED** [review]
Small fix.

File: .github/workflows/conform-pr.yml [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+#
+# * We require approval for PRs from first-time contributors. That's a built-in feature for all actions.
+# * For workflows that checkout source code,
+#   we require the `trust` label to be assigned to PRs by FerretDB maintainers after reviewing changes.
+#   Only a few trusted people have permission to do that.
+# * Thanks to the way `pull_request_target` trigger works,
+#   PR changes in the workflow itself are not run until they are merged.
+# * We use a short-lived automatic `GITHUB_TOKEN` instead of a long-living personal access token (PAT) where possible.
+# * Both `GITHUB_TOKEN`s and PATs have minimal permissions.
+# * We publish Docker images from PRs as a separate packages that should not be run by users.
```
[the remark this answers is not in the document]
Comment (AlekSi) [reply .github/workflows/conform-pr.yml]:
```
# But rogue PR authors could try to steal our secrets.
# We prevent that with the following:
```

That applies to all items

**Review by b1ron — APPROVED** [review]

**Review by Fashander — APPROVED** [review]

**Pull request merged**